todo_app/app.py

```diff
@@ -53,25 +59,28 @@ def new():
     #@role_required('writer')
     def move(cardID, newList):
         if role_required('writer') :
-            exit
+            raise Exception("You dont have write access")
```

Comment:
I can still bypass this bit as before - if we adjust this to match the `new` function then that should fix it:
```python
        if not is_writer():
            raise Exception("You dont have write access")
```

Bonus that if we do switch that over, I think we could then delete the `role_required` function in preference for the `is_writer` function